test/jasmine/tests/waterfall_test.js

```diff
@@ -1424,7 +1486,7 @@ describe('waterfall hover', function() {
             })
             .then(function() {
                 assertHoverLabelContent({
-                    nums: '2.2\n4.4 ▲\nInitial: −2.2',
+                    nums: '2.2\nFinal: 2.2\n4.4 ▲\nInitial: −2.2',
```

Comment:
Hmm. So wait `Final: ` is new?

Reply:
Yes.

Reply:
Was this a requirement anywhere?

Reply:
No. And we could possibly drop that. 
But that make it clear what the value is pointing to.